### ISSUE TITLE
Should `configureScene` be handled by the route instead of the navigator?

### ISSUE THREAD

**Issue by despairblue** [exponentjs/ex-navigator#12]
I have an app with a tabbar, three tabs, one global navigator and one navigator in each tab. All views pushed onto the global navigator should float from the bottom (they hide the tabbar). All views pushed onto one of the other navigators should float from the right (they don't hide the tabbar, it's still possible to navigate to other tabs without popping the views).

Some views can be pushed to any navigator (like a log-in-view). Now I either need to make two different route factories for the login view or pass in props into the route factory to create a route with the specific scene configuration.

Either way the views that push the log-in-view need to know where they are in the view hierarchy to either choose the correct route factory or to pass the correct scene configuration into the route factory. That seems wrong, as it goes against composition. :confused: 

Shouldn't only the place of the navigator in the view hierarchy decide about how the next view is shown?

**Comment by ide** [comment]
Makes sense. We probably want both the navigator and the route to decide what the transition should look like. For example, in your top-level navigator sometimes you may want to float a scene from the top instead of the bottom, depending on the scene.

Would this API work for you?

``` js
<ExNavigator
  defaultSceneConfig={FloatFromBottom}
/>
```

**Comment by despairblue** [comment]
lgtm :+1:

**Comment by cancan101** [comment]
It seems what is there is just a bug? [RN shows](https://facebook.github.io/react-native/docs/navigator.html#configurescene) on the Navigator.
